internal/native/mock_server.go

```diff
@@ -555,8 +397,8 @@ func (m *MockServer) UsingPlugin(pluginName string, pluginVersion string) error
 	cPluginVersion := C.CString(pluginVersion)
 	defer free(cPluginVersion)
 
-	InstallSignalHandlers()
 	r := C.pactffi_using_plugin(m.pact.handle, cPluginName, cPluginVersion)
+	InstallSignalHandlers()
```

Comment:
we need to install our signal handlers after starting our plugins. I wondered why we were still getting flakes after I had got it passing consistently first time before, and I went back and looked at my previous changes I'd tested and did a 😅